src/models/Question.js

```diff
@@ -0,0 +1,17 @@
+import query from '../db/index';
+
+exports.queryGetQuestions = async (productId) => {
+  const { rows } = await query(`SELECT * FROM questions WHERE product_id = ${productId}`);
+
+  return rows;
+};
+
+exports.queryAddQuestion = async ({
```

Comment:
May want to add some error handling here with a 'try/catch' (for all of these database queries). For example, what happens to the request if the database is offline?

Reply:
Thanks. I will look into that. This wasn't supposed to be part of this PR but I got ahead.